sentinelone/plugin.spec.yaml

```diff
@@ -147,371 +147,692 @@ types:
     size:
       title: Size
       description: Size of installed application
-      type: string
+      type: integer
       required: false
     name:
       title: Name
       description: Name of installed application
       type: string
       required: false
-  agents_processes:
-    startTime:
-      title: Start time
-      description: Start time
+  pagination:
+    totalItems:
+      title: Total Items
+      type: integer
+      description: Total items
+      required: false
+    nextCursor:
+      title: Next Cursor
       type: string
+      description: Next cursor
       required: false
-    processName:
-      title: Process name
-      description: Process name
+  agentDetectionInfo:
+    accountId:
+      title: "Accountid"
```

Comment:
These need to be titleized. yes...All of them.

Reply:
This is the thing I hate most about big complex output types.

Reply:
Because this is already a major version bump this makes sense, the customer will need to create a new connection anyway

Reply:
@jmcadams-r7 thanks for shouting this out! Realized that the 2.0 and 2.1 api responses were different, so I reverted the additions from above, forced that trigger to use the 2.0 endpoint and renamed the "data" custom type to "threat_data" for clarity